config.json

```diff
@@ -872,9 +872,13 @@
       "slug": "poker",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Domain-specific languages",
+        "Equality",
+        "Games",
+        "Parsing",
+        "Strings"
```

Comment:
What do you think about "Pattern matching", and "Sequences" as candidate topics ?
A poker "straight" is like a fixed size sequence. A full-house is like a pattern 3 of something and 2 of something else.

I agree that Domain-specific languages is stretching it a bit.  There are objects/nouns (the cards), but no actions/verbs for "the language".